#! /usr/bin/env python

PACKAGE = "gohi_gowhere"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Site1_X", double_t, 0, "the x of a kicthen point ", 1.0, -30.0, 30.0)
gen.add("Site1_Y", double_t, 0, "the y of a kicthen point ", 1.0, -30.0, 30.0)
gen.add("Site1_Z", double_t, 0, "the z of a kicthen point ", 1.0, -30.0, 30.0)

gen.add("Site2_X", double_t, 0, "the x of a bedroom point ", 1.0, -30.0, 30.0)
gen.add("Site2_Y", double_t, 0, "the y of a bedroom point ", 1.0, -30.0, 30.0)
gen.add("Site2_Z", double_t, 0, "the z of a bedroom point ", 1.0, -30.0, 30.0)

gen.add("nav_end", bool_t, 0, "Check to start the test", False)


exit(gen.generate(PACKAGE, "simple_set_point", "SimpleSetPoint"))
